### PR TITLE
[Elastic Log Driver] Fix file check bug, deal with logging

### PR DESCRIPTION
**Pull request by fearful-symmetry**
This addresses a bug where we weren't properly checking to see if a `meta.json` file existed, and also a default debug log level. Users can now config the log level with `LOG_DRIVER_LEVEL`

### CODE REVIEW

File: x-pack/dockerlogbeat/config.json [hunk 1]
```diff
@@ -10,6 +10,14 @@
...
+      {
+        "description": "debug level",
+        "name": "LOG_DRIVER_LEVEL",
+        "value": "info",
+        "Settable": [
+          "value"
+        ]
+      },
       {
         "description": "libbeat env hack",
```
Comment (jsoriano) [inline x-pack/dockerlogbeat/config.json]: :smile:

File: x-pack/dockerlogbeat/pipelinemanager/pipelineManager.go [hunk 2]
```diff
@@ -43,8 +43,7 @@ type PipelineManager struct {
 // NewPipelineManager creates a new Pipeline map
 func NewPipelineManager(logCfg *common.Config) *PipelineManager {
 	return &PipelineManager{
-		Logger: logp.NewLogger("PipelineManager"),
-		//mu:        new(sync.Mutex),
```
Comment (jsoriano) [inline x-pack/dockerlogbeat/pipelinemanager/pipelineManager.go]: :+1:

File: x-pack/dockerlogbeat/handlers.go [hunk 3]
```diff
@@ -70,7 +70,7 @@ func stopLoggingHandler(pm *pipelinemanager.PipelineManager) func(w http.Respons
 		go func() {
 			err = pm.CloseClientWithFile(stopReq.File)
 			if err != nil {
-				pm.Logger.Infof(" Got stop request error %#v\n", err)
+				pm.Logger.Errorf(" Got stop request error %#v\n", err)
```
Comment (jsoriano) [inline x-pack/dockerlogbeat/handlers.go]: Is this space intended?
```suggestion
				pm.Logger.Errorf("Got stop request error %#v\n", err)
```

File: x-pack/dockerlogbeat/main.go [hunk 4]
```diff
@@ -26,8 +26,12 @@ import (
 
 // genNewMonitoringConfig is a hacked-in function to enable a debug stderr logger
 func genNewMonitoringConfig() (*common.Config, error) {
+	lvl, isSet := os.LookupEnv("LOG_DRIVER_LEVEL")
```
Comment (kaiyan-sheng) [inline x-pack/dockerlogbeat/main.go]: no need to change 😬 maybe logDriverLevel or ldl is better?

**Comment by mikemadden42** [comment]
run beats-ci/package

**Comment by mikemadden42** [comment]
run beats-ci/package

**Comment by mikemadden42** [comment]
run beats-ci/package

**Comment by mikemadden42** [comment]
run beats-ci/package

**Pull request merged**